main.scss

```diff
@@ -13,6 +13,8 @@
 	@import "src/scss/print";
 	@import "src/scss/search";
 	@import "src/scss/tools";
+	@import "src/scss/themes/dark-on-light";
```

Comment:
Basically, initially, if you look at base.scss, there was a class `light-on-dark`, so I was going to keep that, and create `dark-on-light` which are more explicit names. I ended up making `light-on-dark` the default, instead of no default and removing that class. So basically the only reason left that class naming is explicitness, but if most people prefer just `light`, I'm happy to go with that